azerothcore/azerothcore-wotlk
# fix (DB/conditions): Drain Schematics

Update of Drain Schematics condition drop to completion of a quest As the Crow Flies (9718).



## Changes Proposed:
-  Update conditions of drop of Drain Schematics
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/13617

## SOURCE:


## Tests Performed:

- I tested this in-game accordingly steps to reproduce in linked issue
- 


## How to Test the Changes:


Steps are taken from linked issue
1.  .quest add 9718
2. .quest complete 9718
3.  .go c id 17841
4.  .go c id 18340
5. Kill Steam Pump Overseers
6. Loot and see Drain Schematics dropped

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
